doc-examples/r2dbc-example-groovy/build.gradle

```diff
@@ -23,13 +23,14 @@ micronaut {
 
 dependencies {
     compileOnly projects.micronautDataProcessor
+    compileOnly mnSerde.micronaut.serde.api
     implementation projects.micronautDataR2dbc
     implementation mnSql.micronaut.jdbc.hikari
     implementation(libs.managed.jakarta.transaction.api)
 
     runtimeOnly mnR2dbc.r2dbc.postgresql
     runtimeOnly mnSql.postgresql
 
+    testImplementation mnSerde.micronaut.serde.api
```

Comment:
`compileTestGroovy` didn't work without this with new gradle plugin, existing `compileOnly` was not enough